PyDSS/pyPostprocessor/PostprocessScripts/AutomatedVoltageUpgrade.py

```diff
@@ -539,7 +553,8 @@ def __init__(self, project, scenario, inputs, dssInstance, dssSolver, dssObjects
                 # determining key with minimum objective func. at various levels
                 # (at this point includes pre-reg, sub-LTC)
                 min_cluster = ''
-                min_severity = pow(len(self.all_bus_names), 2) * len(self.config["tps_to_test"]) * self.upper_limit
+                min_severity = 1000000000
+                # min_severity = pow(len(self.all_bus_names), 2) * len(self.config["tps_to_test"]) * self.upper_limit

```

Comment:
Is there something wrong with this logic? I suggest adding a comment about why this is disabled.